src/modules/location/index.ts

```diff
@@ -140,7 +140,7 @@ export class LocationModule {
   }
 
   /**
-   * Generates a random fictional city name for the locale.
+   * Generates a random fictional or real city name for the locale.
```

Comment:
@ST-DDT 
Do you like this?

```suggestion
   * Generates a random localized city name.
```

Reply:
I think for now it is okay.
We can think about better descriptions at a later point in time/different PR.